### PR TITLE
feat: add Keptn 0.11.2

### PR DESCRIPTION
**Pull request by thisthat**
Signed-off-by: Giovanni Liva <giovanni.liva@dynatrace.com>

Preview: https://deploy-preview-973--keptn.netlify.app/

### CODE REVIEW

File: content/docs/0.11.x/continuous_delivery/deployment_helm/index.md [hunk 1]
```diff
@@ -30,12 +30,12 @@ carts   1/1     1            1           56m   carts        docker.io/keptnexamp
 
 When triggering a delivery (with a new artifact), we are updating the values.yaml file in the Helm Charts with the respective image name.
 
-* [chart/values.yaml](https://github.com/keptn/examples/blob/0.11.0/onboarding-carts/carts/values.yaml#L1)
-* [chart/templates/deployment.yaml](https://github.com/keptn/examples/blob/0.11.0/onboarding-carts/carts/templates/deployment.yaml#L22)
+* [chart/values.yaml](https://github.com/keptn/examples/blob/0.11.2/onboarding-carts/carts/values.yaml#L1)
+* [chart/templates/deployment.yaml](https://github.com/keptn/examples/blob/0.11.2/onboarding-carts/carts/templates/deployment.yaml#L22)
```
Comment (oleg-nenashev) [inline content/docs/0.11.x/continuous_delivery/deployment_helm/index.md:34]: These links tags do not exist in the examples repo

File: content/docs/0.11.x/operate/advanced_install_options/index.md [hunk 2]
```diff
@@ -12,15 +12,15 @@ Therefore, the [helm CLI](https://helm.sh) is required to execute of the followi
...
 ```
-helm install keptn https://github.com/keptn/keptn/releases/download/0.11.0/keptn-0.11.0.tgz -n keptn --create-namespace --wait --set=continuous-delivery.enabled=true,control-plane.apiGatewayNginx.type=LoadBalancer
+helm install keptn https://github.com/keptn/keptn/releases/download/0.11.2/keptn-0.11.2.tgz -n keptn --create-namespace --wait --set=continuous-delivery.enabled=true,control-plane.apiGatewayNginx.type=LoadBalancer
 ```
 
 * *Install Keptn execution-plane:*
 
 ```
-helm install jmeter-service https://github.com/keptn/keptn/releases/download/0.11.0/jmeter-service-0.11.0.tgz -n keptn --create-namespace --wait
+helm install jmeter-service https://github.com/keptn/keptn/releases/download/0.11.2/jmeter-service-0.11.2.tgz -n keptn --create-namespace --wait
```
Comment (oleg-nenashev) [inline content/docs/0.11.x/operate/advanced_install_options/index.md:21]: `0.11.2` tag does not exist yet

**Review by oleg-nenashev — COMMENTED** [review]
It looks like something is wrong with versions

* https://github.com/keptn/examples/ does not have 0.11.0+ tags yet
* 0.11.2 does not exist in this repo. Maybe as designed, but the R title suggests otherwise

File: content/docs/0.11.x/continuous_delivery/deployment_helm/index.md [hunk 3]
```diff
@@ -5,18 +5,18 @@ weight: 10
...
-Keptn uses [Helm v3](https://helm.sh/) for deploying a services to a Kubernetes cluster. This is currently implemented in the [helm-service](https://github.com/keptn/keptn/tree/0.11.0/helm-service).
+Keptn uses [Helm v3](https://helm.sh/) for deploying a services to a Kubernetes cluster. This is currently implemented in the [helm-service](https://github.com/keptn/keptn/tree/0.11.2/helm-service).
 Keptn's helm-service supports the following deployment strategies:
 
 * **Direct deployments**
 * **Blue-green deployments**
 * **user-managed deployments** (experimental)
 
-The explanation below is based on the provided Helm Chart for the carts microservice, see [Charts](https://github.com/keptn/examples/tree/0.11.0/onboarding-carts/carts) for details.
+The explanation below is based on the provided Helm Chart for the carts microservice, see [Charts](https://github.com/keptn/examples/tree/0.10.0/onboarding-carts/carts) for details.
```
Comment (johannes-b) [inline content/docs/0.11.x/continuous_delivery/deployment_helm/index.md:15]: Can we please release: https://github.com/keptn/examples with version 0.11. to keep it aligned with the Keptn releases?

**Review by oleg-nenashev — APPROVED** [review]
Is there a follow-up ticket for Keptn examples update somewhere?

File: content/docs/0.11.x/continuous_delivery/deployment_helm/index.md [hunk 4]
```diff
@@ -25,17 +25,17 @@ $ kubectl get deployments -n sockshop-dev carts -owide
 
 ```
 NAME    READY   UP-TO-DATE   AVAILABLE   AGE   CONTAINERS   IMAGES                                 SELECTOR
-carts   1/1     1            1           56m   carts        docker.io/keptnexamples/carts:0.11.1   app=carts
```
Comment (johannes-b) [inline content/docs/0.11.x/continuous_delivery/deployment_helm/index.md:28]: Please revert the change. The carts service is used in version `0.11.1` (see here: https://github.com/keptn-sockshop/carts/tree/release-0.11.0)

File: content/docs/0.11.x/continuous_delivery/deployment_helm/index.md [hunk 5]
```diff
@@ -25,17 +25,17 @@ $ kubectl get deployments -n sockshop-dev carts -owide
...
+carts   1/1     1            1           56m   carts        docker.io/keptnexamples/carts:0.10.0   app=carts
 ```
 
 When triggering a delivery (with a new artifact), we are updating the values.yaml file in the Helm Charts with the respective image name.
 
-* [chart/values.yaml](https://github.com/keptn/examples/blob/0.11.0/onboarding-carts/carts/values.yaml#L1)
-* [chart/templates/deployment.yaml](https://github.com/keptn/examples/blob/0.11.0/onboarding-carts/carts/templates/deployment.yaml#L22)
+* [chart/values.yaml](https://github.com/keptn/examples/blob/0.10.0/onboarding-carts/carts/values.yaml#L1)
```
Comment (johannes-b) [inline content/docs/0.11.x/continuous_delivery/deployment_helm/index.md:33]: Depending on the release of https://github.com/keptn/examples

File: content/docs/0.11.x/continuous_delivery/deployment_helm/index.md [hunk 6]
```diff
@@ -25,17 +25,17 @@ $ kubectl get deployments -n sockshop-dev carts -owide
...
+* [chart/templates/deployment.yaml](https://github.com/keptn/examples/blob/0.10.0/onboarding-carts/carts/templates/deployment.yaml#L22)
 
 ## Blue-green deployments
 
-If the deployment strategy of a stage in the [shipyard](https://github.com/keptn/examples/blob/0.11.0/onboarding-carts/shipyard.yaml) is configured as `deploymentstrategy: blue_green_service`, Helm creates two
+If the deployment strategy of a stage in the [shipyard](https://github.com/keptn/examples/blob/0.10.0/onboarding-carts/shipyard.yaml) is configured as `deploymentstrategy: blue_green_service`, Helm creates two
```
Comment (johannes-b) [inline content/docs/0.11.x/continuous_delivery/deployment_helm/index.md:38]: Depending on the release of https://github.com/keptn/examples

File: content/docs/0.11.x/continuous_delivery/deployment_helm/index.md [hunk 7]
```diff
@@ -45,17 +45,17 @@ $ kubectl get deployments -n sockshop-staging carts carts-primary -owide
 
 ```
 NAME            READY   UP-TO-DATE   AVAILABLE   AGE   CONTAINERS   IMAGES                                 SELECTOR
-carts-primary   1/1     1            1           56m   carts        docker.io/keptnexamples/carts:0.11.1   app=carts-primary
-carts           0/0     0            0            3m   carts        docker.io/keptnexamples/carts:0.11.2   app=carts
+carts-primary   1/1     1            1           56m   carts        docker.io/keptnexamples/carts:0.10.0   app=carts-primary
```
Comment (johannes-b) [inline content/docs/0.11.x/continuous_delivery/deployment_helm/index.md:48]: Please revert the change (see above)

File: content/docs/0.11.x/continuous_delivery/deployment_helm/index.md [hunk 8]
```diff
@@ -45,17 +45,17 @@ $ kubectl get deployments -n sockshop-staging carts carts-primary -owide
...
 ```
 
 
 When triggering a delivery (with a new artifact, e.g., 0.11.2), a canary deployment will be modified and scaled up.
 
 ```
 NAME            READY   UP-TO-DATE   AVAILABLE   AGE   CONTAINERS   IMAGES                                 SELECTOR
-carts-primary   1/1     1            1           57m   carts        docker.io/keptnexamples/carts:0.11.1   app=carts-primary
-carts           0/1     1            1            1m   carts        docker.io/keptnexamples/carts:0.11.2   app=carts
+carts-primary   1/1     1            1           57m   carts        docker.io/keptnexamples/carts:0.10.0   app=carts-primary
```
Comment (johannes-b) [inline content/docs/0.11.x/continuous_delivery/deployment_helm/index.md:57]: revert

File: content/docs/0.11.x/continuous_delivery/deployment_helm/index.md [hunk 9]
```diff
@@ -64,17 +64,17 @@ Once testing has finished, the primary deployment is upgraded to the new version
 
 ```
 NAME            READY   UP-TO-DATE   AVAILABLE   AGE   CONTAINERS   IMAGES                                 SELECTOR
-carts-primary   1/1     1            1            3m   carts        docker.io/keptnexamples/carts:0.11.2   app=carts-primary
-carts           1/1     1            1            1d   carts        docker.io/keptnexamples/carts:0.11.2   app=carts
```
Comment (johannes-b) [inline content/docs/0.11.x/continuous_delivery/deployment_helm/index.md:68]: revert

File: content/docs/0.11.x/continuous_delivery/deployment_helm/index.md [hunk 10]
```diff
@@ -64,17 +64,17 @@ Once testing has finished, the primary deployment is upgraded to the new version
...
+carts           1/1     1            1            1d   carts        docker.io/keptnexamples/carts:0.10.0   app=carts
 ```
 
 After a new pod for the primary deployment has been successfully deployed, traffic is shifted to the primary deployment
  and the canary deployment is scaled down:
 
 ```
 NAME            READY   UP-TO-DATE   AVAILABLE   AGE   CONTAINERS   IMAGES                                 SELECTOR
-carts-primary   1/1     1            1            4m   carts        docker.io/keptnexamples/carts:0.11.2   app=carts-primary
-carts           0/0     0            0            1d   carts        docker.io/keptnexamples/carts:0.11.2   app=carts
```
Comment (johannes-b) [inline content/docs/0.11.x/continuous_delivery/deployment_helm/index.md:77]: revert

File: content/docs/0.11.x/continuous_delivery/deployment_helm/index.md [hunk 11]
```diff
@@ -84,7 +84,7 @@ carts           0/0     0            0            1d   carts        docker.io/ke
 future releases of Keptn.*
 
 
-If the deployment strategy of a stage in the [shipyard](https://github.com/keptn/examples/blob/0.11.0/onboarding-carts/shipyard.yaml)
```
Comment (johannes-b) [inline content/docs/0.11.x/continuous_delivery/deployment_helm/index.md:87]: Depending on the release of https://github.com/keptn/examples

File: content/docs/0.11.x/continuous_delivery/multi_stage/index.md [hunk 12]
```diff
@@ -149,7 +149,7 @@ keptn trigger delivery --project=$PROJECTNAME --service=$SERVICENAME --image=$IM
     "stage": "[STAGENAME]",
     "configurationChange": {
       "values": {
-        "image": "keptn-examples/carts:0.11.3"
```
Comment (johannes-b) [inline content/docs/0.11.x/continuous_delivery/multi_stage/index.md:152]: revert

File: content/docs/0.11.x/manage/service/index.md [hunk 13]
```diff
@@ -33,7 +33,7 @@ After creating a service, you need to provide a Helm Chart for the service to de
 1. The Helm chart _has_ to contain a `values.yaml` file with at least the `image` and `replicaCount` parameter for the deployment. These `image` and `replicaCount` parameters have to be used in the deployment. An example is shown below:
   
   ```yaml
-  image: docker.io/keptnexamples/carts:0.11.1
```
Comment (johannes-b) [inline content/docs/0.11.x/manage/service/index.md:36]: revert

**Comment by thisthat** [comment]
Links have been updated and examples released! Could you please have another look at the PR?

File: content/docs/0.11.x/continuous_delivery/deployment_helm/index.md [hunk 14]
```diff
@@ -54,7 +54,7 @@ When triggering a delivery (with a new artifact, e.g., 0.11.2), a canary deploym
 
 ```
 NAME            READY   UP-TO-DATE   AVAILABLE   AGE   CONTAINERS   IMAGES                                 SELECTOR
-carts-primary   1/1     1            1           57m   carts        docker.io/keptnexamples/carts:0.11.1   app=carts-primary
+carts-primary   1/1     1            1           57m   carts        docker.io/keptnexamples/carts:0.11.2   app=carts-primary
```
Comment (johannes-b) [inline content/docs/0.11.x/continuous_delivery/deployment_helm/index.md:57]: This needs to be `0.11.1`

**Review by johannes-b — APPROVED** [review]
lgtm
